app/models/bookmark.rb

```diff
@@ -1,3 +1,10 @@
+class ControllerValidator < ActiveModel::EachValidator
```

Comment:
It might be a good idea to put this under lib/ or somewhere, instead of in this model.